packages/editor/src/components/ComponentForm/StyleTraitForm/StyleTraitForm.tsx

```diff
@@ -267,6 +267,7 @@ export const StyleTraitForm: React.FC<Props> = props => {
                   mode="css"
                   defaultCode={style}
                   onBlur={v => changeStyleContent(i, v)}
+                  needRerenderAfterMount
```

Comment:
`needRerenderAfterMount` always seems to be undefined. is this expected?

Reply:
Actually, it is `true` if no value is set.

Reply:
Sorry, I got confused